app/views/topical_events/_document_list_from_search_api.html.erb

```diff
@@ -1,7 +1,7 @@
 <% heading ||= nil %>
 
 <% if heading %>
-  <section id="<%= heading.downcase.parameterize %>" class="document-block">
+  <section id="<%= heading.downcase.parameterize %>" class="govuk-!-margin-bottom-7">
```

Comment:
Is there anything we can add to the commit message as to why these are the appropriate spacing? 

There seems a bit of a mental leap from the commit message saying it's appropriate to understanding why - so could be something like "in orgs we have a 30px padding [link to rule] and margin-bottom-7 replicates that"

Reply:
Agree with Kevin. Also it would be useful to have a URL for a page that this impacts.

Reply:
Worth noting that `govuk-!-margin-bottom-7` has a margin of 40px of desktop, but 25px on mobile. All of the override classes are responsive - so will have a different margin depending on screen size.